app/models/card.rb

```diff
@@ -5,6 +5,8 @@ class Card < ActiveRecord::Base
 
   validate :the_text_cannot_be_the_same
 
+  scope :pending, -> { where('review_date >=?', Date.today) }
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.